satellite/audit/service.go

```diff
@@ -46,7 +50,7 @@ type Service struct {
 // NewService instantiates a Service with access to a Cursor and Verifier
 func NewService(log *zap.Logger, config Config, metainfo *metainfo.Service,
 	orders *orders.Service, transport transport.Client, overlay *overlay.Service,
-	containment Containment, identity *identity.FullIdentity) (*Service, error) {
+	containment Containment, identity *identity.FullIdentity, metaLoop *metainfo.Loop) (*Service, error) {
```

Comment:
It seems we don't use this new `metaLoop` param anywhere.

Reply:
Why is that?

Reply:
oops, when separating the two services I must've forgotten to delete this param